openlibrary/templates/login.html

```diff
@@ -16,6 +16,16 @@ <h1>$_("Log In")</h1>
 
   $else:
       <form id="register" class="login olform" name="register" method="post">
+          $if not is_openlibrary_host():
```

Comment:
```suggestion
          $if "dev" in ctx.features:
```
We can remove the need for a new Python function by checking for the "dev" feature flag here.  This is the same check used to determine whether "development version" is overlaid on top of the Open Library logo: https://github.com/internetarchive/openlibrary/blob/10781dbb08e50e1eecc5de6a85909d973601d5c1/openlibrary/templates/site/head.html#L61